docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -32,6 +32,39 @@ Type: UInt32
 
 Default: `120`
 
+## asynchronous_metric_log
```

Comment:
There is already a page for `asynchronous_metric_log` I think that the details in this PR should be added under [operations/system-tables/asynchronous_metric_log ](https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/operations/system-tables/asynchronous_metric_log.md).

Reply:
I added it here as we have a similar setting for `metric_log`. 

There is a system tables page: https://clickhouse.com/docs/en/operations/system-tables/metric_log

And a section in the global server setting to enable/disable them: https://clickhouse.com/docs/en/operations/server-configuration-parameters/settings#metric_log

However I need to add a link to the asynchronous_metric_log server settings in the system-table page.

Reply:
Sure, I think it makes sense to add it in both places.